source/releasenotes/changes.rst

```diff
@@ -13,14 +13,542 @@
+.. _`#3855`: https://github.com/apache/cloudstack/pull/3855
+.. _`#3383`: https://github.com/apache/cloudstack/pull/3383
+.. _`#3819`: https://github.com/apache/cloudstack/pull/3819
+.. _`#3604`: https://github.com/apache/cloudstack/pull/3604
+.. _`#3840`: https://github.com/apache/cloudstack/pull/3840
+.. _`#3834`: https://github.com/apache/cloudstack/pull/3834
+.. _`#3848`: https://github.com/apache/cloudstack/pull/3848
+.. _`#3726`: https://github.com/apache/cloudstack/pull/3726
+.. _`#3846`: https://github.com/apache/cloudstack/pull/3846
+.. _`#3845`: https://github.com/apache/cloudstack/pull/3845
+.. _`#3761`: https://github.com/apache/cloudstack/pull/3761
+.. _`#3758`: https://github.com/apache/cloudstack/pull/3758
+.. _`#3728`: https://github.com/apache/cloudstack/pull/3728
+.. _`#3727`: https://github.com/apache/cloudstack/pull/3727
+.. _`#3477`: https://github.com/apache/cloudstack/pull/3477
+.. _`#3825`: https://github.com/apache/cloudstack/pull/3825
+.. _`#3759`: https://github.com/apache/cloudstack/pull/3759
+.. _`#3806`: https://github.com/apache/cloudstack/pull/3806
+.. _`#3795`: https://github.com/apache/cloudstack/pull/3795
+.. _`#3776`: https://github.com/apache/cloudstack/pull/3776
+.. _`#3778`: https://github.com/apache/cloudstack/pull/3778
+.. _`#3743`: https://github.com/apache/cloudstack/pull/3743
+.. _`#3682`: https://github.com/apache/cloudstack/pull/3682
+.. _`#3658`: https://github.com/apache/cloudstack/pull/3658
+.. _`#3662`: https://github.com/apache/cloudstack/pull/3662
+.. _`#3793`: https://github.com/apache/cloudstack/pull/3793
+.. _`#3597`: https://github.com/apache/cloudstack/pull/3597
+.. _`#3721`: https://github.com/apache/cloudstack/pull/3721
+.. _`#3715`: https://github.com/apache/cloudstack/pull/3715
+.. _`#3755`: https://github.com/apache/cloudstack/pull/3755
+.. _`#3729`: https://github.com/apache/cloudstack/pull/3729
+.. _`#3733`: https://github.com/apache/cloudstack/pull/3733
+.. _`#3747`: https://github.com/apache/cloudstack/pull/3747
+.. _`#3754`: https://github.com/apache/cloudstack/pull/3754
+.. _`#3767`: https://github.com/apache/cloudstack/pull/3767
+.. _`#3765`: https://github.com/apache/cloudstack/pull/3765
+.. _`#3617`: https://github.com/apache/cloudstack/pull/3617
+.. _`#3640`: https://github.com/apache/cloudstack/pull/3640
+.. _`#3635`: https://github.com/apache/cloudstack/pull/3635
+.. _`#3636`: https://github.com/apache/cloudstack/pull/3636
+.. _`#3678`: https://github.com/apache/cloudstack/pull/3678
+.. _`#3605`: https://github.com/apache/cloudstack/pull/3605
+.. _`#3612`: https://github.com/apache/cloudstack/pull/3612
+.. _`#3666`: https://github.com/apache/cloudstack/pull/3666
+.. _`#3620`: https://github.com/apache/cloudstack/pull/3620
+.. _`#3648`: https://github.com/apache/cloudstack/pull/3648
+.. _`#3627`: https://github.com/apache/cloudstack/pull/3627
+.. _`#3608`: https://github.com/apache/cloudstack/pull/3608
+.. _`#3574`: https://github.com/apache/cloudstack/pull/3574
```

Comment:
these links are already in the table above, why do we need them again?

Reply:
Nope, not redundant, those are links for the table above. Some weird stuff, dont show when you actually see the docs that ape has built!